packages/ember-routing-htmlbars/tests/helpers/element_action_test.js

```diff
@@ -1002,6 +1085,29 @@ QUnit.test('should respect preventDefault=false option if provided', function()
   equal(event.isDefaultPrevented(), false, 'should not preventDefault');
 });
 
+QUnit.test('should respect preventDefault=false option if provided bound', function() {
+  view = EmberView.create({
```

Comment:
I believe you can use an `Ember.Component` here and set the `targetObject` to be the `controller`. I am wary of adding new tests that use `EmberView` directly.